pkg/envoy/envoy_test.go

```diff
@@ -38,7 +38,7 @@ func (s *EnvoySuite) TestEnvoy(c *C) {
 	stateLogDir := c.MkDir()
 
 	// launch debug variant of the Envoy proxy
-	Envoy := StartEnvoy(9901, stateLogDir, stateLogDir, 42)
```

Comment:
Typo in your email in the comment message

Reply:
Fixed.